ja/views/json-and-xml-views.rst

```diff
@@ -230,7 +230,7 @@ JSONP レスポンスの返すことが出来ます。これに ``true`` を設
             }
 
             // ビューに出力形式をセット
-            $this->viewBuilder()->className($formats[$format]);
+            $this->viewBuilder()->setClassName($formats[$format]);
```

Comment:
follow it

- replaced className() with setClassName(): https://github.com/cakephp/docs/pull/6396